spec/oas_v1.yaml

```diff
@@ -466,6 +518,266 @@ components:
+          items:
+            type: string
+        deployedContracts:
+          summary: Deployed contracts for the project
+          description: >
+            Information about the contracts deployed by the project, including the contract address, chain ID, deployer, and creation block.
+          type: array
+          items:
+            allOf:
+            - $ref: "#/components/schemas/Contract"
+            - type: object
+              properties:
+                verificationProof:
+                  summary: Proof of contract verification
+                  description: Verification of ownership of contract on chain.
+                  type: string
+                openSourceObserverSlug:
+                  summary: Open source observer verification slug 
+                  description: > 
+                    Slug for the open source observer external verification.
+                  type: string
+        categories: 
+          type: array 
+          items:
+            $ref: "#/components/schemas/RetroFundingCategory"
+        funding: 
+          type: object
+          properties:
+            ventureCapital:
+              $ref: "#/components/schemas/FundingInfo"
+            grants:
+              $ref: "#/components/schemas/FundingInfo"
+            optimismGrants:
+              allOf:
+                - $ref: "#/components/schemas/FundingInfo"
+                - type: object
+                  properties:
+                    link:
+                      type: string
+                    type: 
+                      $ref: "#/components/schemas/OptimismGrantType"
+    SocialLinks:
+      summary: Social media links for a project
+      description: >
+        Social media links for a project, including links to Twitter, Discord, and other platforms.
+      type: object
+      properties:
+        twitter:
+          type: string
+        farcaster:
+          type: string
+        mirror:
+          type: string
+        website:
+          type: string
+    OptimismGrantType:
+      summary: Type of Optimism grant
+      description: Enum description of the type of Optimism grant.
+      type: string
+      enum:
+        - DEVELOPMENT
+        - RESEARCH
+        - COMMUNITY
+        - ECOSYSTEM
+        - OTHER
+    FundingInfo:
+      summary: Information about funding for a project
+      description: >
+        Information about funding for a project, including the amount and source.
+      type: object
+      properties:
+        amount:
+          type: string
+        source:
+          type: string
+        date: 
+          type: string
+          format: date-time
+        details:
+          type: string
+    RetroFundingBallotAllocation:
+      summary: A vote for an RetroFunding ballot
+      description: >
+        A vote for an RetroFunding ballot, including information about the vote.
+      type: object
+      properties:
+        metricId:
+          type: integer
+        categoryId:
```

Comment:
a bit nit picking here but we can probably drop `categoryId` -- don't really see how it can be used here